### ISSUE TITLE
Show only specific number of images?

### ISSUE THREAD

**Issue by JasOcher** [fredleblanc/roundabout#92]
When adding more images to a list, it consequently shows more images in the rotator as well.  I really only want it to show 3 images and just hide the rest until it's their turn to come up.  Is there a way to do that?

**Comment by samarulrajt** [comment]
yeah! i too agree with this point when i load images dynamically ...........round about seems not working
